specification/web/resource-manager/Microsoft.Web/stable/2020-09-01/Certificates.json

```diff
@@ -434,6 +434,10 @@
             "canonicalName": {
               "description": "CNAME of the certificate to be issued via free certificate",
               "type": "string"
+            },
+            "domainValidationMethod": {
```

Comment:
Adding a new property or a new api-version to existing GA version is considered as a breaking change. Please use a new api-version.